teamone/settings.py

```diff
@@ -69,9 +69,10 @@
     "allauth.socialaccount",
     "allauth.socialaccount.providers.google",
     "storages",
+    "notifications.apps.NotificationsConfig",
 ]
 
-SITE_ID = 2
```

Comment:
@jackxujh What does the site_id need to be set to on develop? Did you need to change it to get it running locally or is this what the site id should be on the deployment?